From a929e8ef2ec5ad5a64906859a1375b6a3f0771b4 Mon Sep 17 00:00:00 2001
From: Gleb Natapov <gleb@redhat.com>
Date: Tue, 15 Jun 2010 08:11:28 -0300
Subject: [PATCH 2/8] do not advertise hpet to a guest OS.

RH-Author: Gleb Natapov <gleb@redhat.com>
Message-id: <20100615081128.GP21797@redhat.com>
Patchwork-id: 9915
O-Subject: [PATCH SEABIOSv2 RHEL6] do not advertise hpet to a guest OS.
Bugzilla: 602177
RH-Acked-by: Markus Armbruster <armbru@redhat.com>
RH-Acked-by: Jes Sorensen <Jes.Sorensen@redhat.com>
RH-Acked-by: Juan Quintela <quintela@redhat.com>
RH-Acked-by: Amit Shah <amit.shah@redhat.com>

Since we disabled hpet in qemu we must not advertise hpet to a guest OS.

BZ: 602177
Upstream status: the patch is not suitable for upstream since hpet may
                 be enabled there. In upstream we should create HPET ACPI
                 table only when hpet is enabled in qemu. Working on the
                 fix.

Signed-off-by: Gleb Natapov <gleb@redhat.com>
---
 v1->v2:
  - do not remove code to generate HPET tables, only call to the
    function.

--
			Gleb.

Signed-off-by: Eduardo Habkost <ehabkost@redhat.com>
---
 src/acpi.c |    1 -
 1 files changed, 0 insertions(+), 1 deletions(-)

diff --git a/src/acpi.c b/src/acpi.c
index 18830dc..0a368a4 100644
--- a/src/acpi.c
+++ b/src/acpi.c
@@ -623,7 +623,6 @@ acpi_bios_init(void)
     ACPI_INIT_TABLE(build_fadt(bdf));
     ACPI_INIT_TABLE(build_ssdt());
     ACPI_INIT_TABLE(build_madt());
-    ACPI_INIT_TABLE(build_hpet());
     ACPI_INIT_TABLE(build_srat());
 
     u16 i, external_tables = qemu_cfg_acpi_additional_tables();
-- 
1.7.3.2

